src/views/microworldshomepage/microworldshomepage.jsx

```diff
@@ -0,0 +1,72 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+
+var MasonryGrid = require('../../components/masonrygrid/masonrygrid.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var TTTTile = require('../../components/ttt-tile/ttt-tile.jsx');
+var Tiles = require('./ttt.json');
+
+var classNames = require('classnames');
+
+require('../thingstotry/thingstotry.scss');
```

Comment:
If you would like to re-use another view's CSS, then that leads me to think we need another type of component to represent those styles, which we would then use in both views.  Ideally each view only will contain the CSS specific to it, and other common CSS should be contained in reusable components.